src/components/VersionList/VersionList.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from 'classnames';
+import {Mutation} from 'react-apollo';
+import {FILE_DOWNLOAD_URL} from '../../state/mutations';
+import {KF_STUDY_API} from '../../common/globals';
+import {Button, Avatar, Icon} from 'kf-uikit';
+import TimeAgo from 'react-timeago';
+import {fileTypeDetail, formatFileSize} from '../../common/fileMeta';
+/**
+ * Displays unordered studies in grid view (include empty stage message)
+ */
+
+const VersionList = ({className, studyId, fileNode}) => {
+  let versionListClass = classes('FileVersionList', className);
+  const ext = fileNode.name.substr(fileNode.name.lastIndexOf('.'));
+  return (
+    <div className="lg:cell-10 md:cell-9 cell-12">
+      <div className="bg-lightGrey p-8 flex justify-between">
+        <p className="FileInfo--Title">
+          Versions
+          <span className="font-light ml-4">
+            ({fileNode.versions.edges.length})
+          </span>
+        </p>
+        <Button onClick={e => e.preventDefault()}>
+          <img
+            src="https://img.icons8.com/material-sharp/32/000000/upload-to-cloud.png"
+            alt="upload"
+            className="w-4 mr-4"
+          />
+          UPLOAD VERSION
+        </Button>
+      </div>
+      <ul className={versionListClass}>
+        {fileNode.versions.edges.length ? (
+          fileNode.versions.edges
+            .sort((a, b) =>
+              a.node.createdAt < b.node.createdAt
+                ? 1
+                : b.node.createdAt < a.node.createdAt
+                ? -1
+                : 0,
+            )
+            .map(({node}, index) => (
+              <Mutation mutation={FILE_DOWNLOAD_URL} key={node.kfId}>
+                {downloadFile => (
+                  <li key={node.kfId} className="FileVersionList--Element">
+                    <Icon
```

Comment:
this may be a shortcoming of the Icons in the design system but the icons need to match the colors in the mockup